specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-01-preview/images.json

```diff
@@ -0,0 +1,348 @@
+{
+   "swagger": "2.0",
+   "info": {
+      "title": "Cognitive Services API",
+      "description": "Cognitive Services API to support OpenAI Dall-E",
+      "version": "2023-04-01-preview"
+   },
+   "host": "eastus.api.cognitive.microsoft.com",
+   "basePath": "/dalle",
+   "schemes": ["https"],
+   "securityDefinitions": {
+      "Api-Key": {
+        "type": "apiKey",
+        "name": "Api-Key",
+        "in": "header",
+        "description": "Provide your Cognitive Services Azure OpenAI account key here."
+      }
+   },
+   "security": [
+      {
+        "Api-Key": []
+      }
+    ],
+   "parameters": {
+      "apiVersionQueryParameter": {
+        "in": "query",
+        "name": "api-version",
+        "description": "The requested API version.",
+        "required": true,
+        "type": "string"
+      }
+   },
```

Comment:
"parameters": {
    "endpoint": {
      "in": "path",
      "name": "endpoint",
      "description": "Supported Cognitive Services endpoints (protocol and hostname, for example: [https://aoairesource.openai.azure.com](https://aoairesource.openai.azure.com). Replace \"aoairesource\" with your Azure OpenAI account name).",
      "required": true,
      "type": "string",
      "format": "url",
      "x-ms-skip-url-encoding": true,
      "x-ms-parameter-location": "client"
    },
    "apiVersionQueryParameter": {
      "in": "query",
      "name": "api-version",
      "description": "The requested API version.",
      "required": true,
      "type": "string",
      "x-ms-client-default": "2022-12-01",
      "x-ms-parameter-location": "client"
    }
  },

Reply:
I am actually not sure, if we have the endpoint parameter? Right now, our PROD endpoints look something like this: https://eastus.api.cognitive.microsoft.com/dalle/... or would we automatically have this url "https://aoairesource.openai.azure.com/" when the Azure OpenAI resource is created?